pkg/blockdeviceclaim/v1alpha1/kubernetes.go

```diff
@@ -241,3 +255,44 @@ func (k *Kubeclient) Delete(name string, deleteOpts *metav1.DeleteOptions) error
 	}
 	return k.del(cli, k.namespace, name, deleteOpts)
 }
+
+// Patch patches the block device claim if present in kubernetes cluster
+func (k *Kubeclient) Patch(name string, pt types.PatchType, data []byte, subresources ...string) (*apis.BlockDeviceClaim, error) {
+	if len(name) == 0 {
+		return nil, errors.New("failed to patch block device claim: missing bdc name")
+	}
+	cli, err := k.getClientsetOrCached()
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to patch bdc: {%s}", name)
+	}
+	return k.patch(cli, k.namespace, name, pt, data, subresources...)
+}
+
+// PatchBDCWithLabel patches the block device claim with provided labels
+func (k *Kubeclient) PatchBDCWithLabel(label map[string]string, bdcObj *apis.BlockDeviceClaim) error {
+	BuildObj, err := BuilderForAPIObject(bdcObj).
+		WithLabels(label).Build()
+	if err != nil {
+		return errors.Wrapf(err, "failed to build bdc {%s}", bdcObj.Name)
+	}
+	bdcBytes, err := json.Marshal(BuildObj)
+	_, err = k.Patch(bdcObj.Name, types.MergePatchType, bdcBytes)
+	if err != nil {
+		return errors.Wrapf(err, "failed to patch bdc {%s}", bdcObj.Name)
+	}
+	return nil
+}
+
+// DeleteMultiPleBDCs deletes list of block device claim
+func (k *Kubeclient) DeleteMultipleBDCs(bdcList []string) error {
```

Comment:
Why are we not using DeleteCollection

Reply:
@AmitKumarDas I am removing unwanted things from this PR. I went on different on assumption just give few minutes I will push changes